### PR TITLE
py3: fix AttributeError with urlopen

### PR DESCRIPTION
**Pull request by MartinBasti**
Py3 has urlopen in urllib.requests module, py2 has urlopen in urllib

Fixes: #75

### CODE REVIEW

File: Lib/ldap/schema/subentry.py [hunk 1]
```diff
@@ -474,8 +474,11 @@ def urlfetch(uri,trace_level=0,bytes_mode=None):
     l.unbind_s()
     del l
   else:
-    import urllib,ldif
-    ldif_file = urllib.urlopen(uri)
+    import ldif
+    from ldap.compat import urlopen
+    if not uri.lower().startswith("file://"):
```
Comment (tiran) [inline Lib/ldap/schema/subentry.py]: That's probably not right. A LDAP schema url could be a http or https link, too. The application should pass in a correct URI.

Comment (tiran) [reply Lib/ldap/schema/subentry.py]: how about:

```
if '://' not in uri:
    import warnings
    warnings.warn("File URI must start with file://", category=DeprecationWarning)
    uri = 'file://{}'.format(uri)
ldap_file = urlopen(uri)

Comment (MartinBasti) [reply Lib/ldap/schema/subentry.py]: It worked with python2, so backward compatibility. And according the pyldap code, the code itself expects just filename, I'm not sure if we should issue a warning

Comment (MartinBasti) [reply Lib/ldap/schema/subentry.py]: EDIT: I wrote nonsense, ignore

Comment (MartinBasti) [reply Lib/ldap/schema/subentry.py]: The longer I'm thinking about it, the more I like that warning actually

**Comment by encukou** [comment]
I've pushed the first change, that was definitely a bug. I've opened https://github.com/pyldap/pyldap/issues/79 to track writing tests for it.

As for the second change, I'm not sure pyldap is the right level for this – perhaps it should go into python-ldap, or be handled in users' code as part of the transition to Python 3. (There aren't that many users of urlfetch, after all.)

**Comment by MartinBasti** [comment]
@encukou well I wanted to keep backward compatibility with this change, but I don't need this, I plan to use `file://` prefix in my code anyway

**Comment by encukou** [comment]
This was merged as 378b13c8361a12dc655146f2cd91f28bbe257599

**Pull request closed without merging**